payment/pom.xml

```diff
@@ -63,6 +63,10 @@
             <groupId>org.springframework.integration</groupId>
             <artifactId>spring-integration-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
```

Comment:
This dependency is configured in the parent pom.xml, so you don't need to reconfigure here.